app/api/routers/breeze_buddy/templates/handlers.py

```
@@ -40,6 +43,48 @@
+    from a client that predates observers can't wipe a populated row.
+    """
+    if not configurations or "observers" not in configurations:
+        return
+
+    try:
+        await upsert_observer_evaluation_config(
+            template_id=template_id,
+            configuration={"evaluators": configurations.get("observers") or []},
+            enabled=True,
+        )
+    except Exception:
```

Ruff (0.16.1)

[warning] Do not catch blind exception: `Exception`

(BLE001)